libs/blocks/carousel/carousel.js

```diff
@@ -20,7 +20,7 @@ function decorateNextPreviousBtns() {
   const previousBtn = createTag(
     'button',
     {
-      class: 'carousel-button carousel-previous',
+      class: 'carousel-button carousel-previous is-delayed',
```

Comment:
There's a lot of repetition for `is-delayed`, would it be worth to turn it into a constant?